cpp/src/Ice/ConnectionI.cpp

```diff
@@ -2822,6 +2828,16 @@ Ice::ConnectionI::sendMessage(OutgoingMessage& message)
         return AsyncStatusQueued;
     }
 
+    // A message is always at least 14 bytes: the header size.
```

Comment:
> Could it be at rest even though it has all these outstanding _sendStreams?

I guess it could in the unlikely scenario that `_writeStream` contains a connection validation message.

> And then we send/queue a new request but don't cancel the inactivity timer task since we just queue the message?

Yes